packages/semantic-ui/src/UpDownWidget/UpDownWidget.js

```diff
@@ -9,22 +8,29 @@ const { getDisplayLabel } = utils;
 function UpDownWidget(props) {
   const {
     id,
+    name,
+    label,
+    value,
     required,
     readonly,
     disabled,
-    label,
-    name,
-    value,
-    options,
     onChange,
     onBlur,
     onFocus,
     autofocus,
-    formContext,
+    options,
     schema,
     uiSchema,
+    formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps(
+    { formContext, options,
+      uiSchema,
+      defaultSchemaProps: {
+        fluid: true,
+        inverted: false,
```

Comment:
Consider moving `options` to a separate line to be consistent with most of the rest of the code. Also, consider not passing `defaultSchemaProps` since it has the same props set as the default value when it is not provided.